src/botPage/bot/TradeEngine/Ticks.js

```diff
@@ -31,18 +31,33 @@ export default Engine =>
                 tickListenerKey = key;
             }
         }
-        getTicks() {
+        getTicks(toString = false) {
             return new Promise(resolve =>
-                this.$scope.ticksService
-                    .request({ symbol: this.symbol })
-                    .then(ticks => resolve(ticks.map(o => o.quote)))
+                this.$scope.ticksService.request({ symbol: this.symbol }).then(ticks =>
+                    resolve(
+                        ticks.map(o => {
+                            if (toString) {
+                                return o.quote.toFixed(this.getPipSize());
+                            } 
+                            return o.quote;
+                            
+                        })
+                    )
+                )
             );
         }
-        getLastTick(raw) {
+        getLastTick(raw, toString = false) {
             return new Promise(resolve =>
                 this.$scope.ticksService
                     .request({ symbol: this.symbol })
-                    .then(ticks => resolve(raw ? getLast(ticks) : getLast(ticks).quote))
+                    .then(ticks => raw ? getLast(ticks) : getLast(ticks).quote)
```

Comment:
We don't need to create another `then` here, we can add the logic in the same statement.
```JavaScript
.then(ticks => {
    let lastTick = raw ? getLast(ticks) : getLast(ticks).quote;
    if (toString && !raw) {
        lastTick = lastTick.toFixed(this.getPipSize());
    }
    resolve(lastTick);
} 
```